src/command_modules/azure-cli-netappfiles/HISTORY.rst

```diff
@@ -0,0 +1,11 @@
+.. :changelog:
+
+Release History
+===============
+
+
+1.0.0 (2019-05-01)
+++++++++++++++++++
+
+* GA release.
+* Previously existed as cli-extension.
```

Comment:
You will need to schedule a full command review to move this from an extension to a module. Do not put the date in the version. "Previously existed as cli-extension" is not a customer-facing thing.

Reply:
> You will need to schedule a full command review to move this from an extension to a module. Do not put the date in the version. "Previously existed as cli-extension" is not a customer-facing thing.

Ok. I will remove the date and the cli-extension text. Can you tell me the process for the command review? I know one of these was already performed, initiated through email with Shane Mainali among others, and the commands agreed in a meeting. It was then decided that because the swagger was in preview that the service should be an extension. I did not expect another review was needed.